source/slang/slang-options.cpp

```diff
@@ -1421,9 +1424,6 @@ struct OptionsParser
         //
         for(auto& rawOutput : rawOutputs)
         {
-            // TODO(DG): As noted below, change support for output formats
-            // with multiple entry points
-
             // For now, all output formats need to be tightly bound to
```

Comment:
There's a comment here about how "down the road" we will need to allow output formats to support multiple entry points. That comment should probably be updated to reflect the new reality.